src/wolfsftp.c

```diff
@@ -1210,40 +1208,18 @@ static int wolfSSH_SFTP_RecvRealPath(WOLFSSH* ssh, int reqId, byte* data,
                 return WS_BAD_FILE_E;
             }
         #endif
-        }
-        WSTRNCAT(wd, "/", WOLFSSH_MAX_FILENAME);
-        WSTRNCAT(wd, r, WOLFSSH_MAX_FILENAME);
-        WMEMCPY(r, wd, WOLFSSH_MAX_FILENAME);
+        ret = wolfSSH_RealPath(wd, r, s, sizeof s);
```

Comment:
So in the non-Windows case we give a buffer of 0s (wd is memset to 0s) as the default path?

Reply:
I see what I didn't do here. This worked with the echoserver because the echoserver is always setting the default path. Updating.